en/deploy-on-gcp-gke.md

```diff
@@ -57,16 +65,75 @@ gcloud config set compute/region <gcp-region>
     {{< copyable "shell-regular" >}}
 
     ```shell
-    gcloud container node-pools create pd --cluster tidb --machine-type n1-standard-4 --num-nodes=1 \
+    gcloud container node-pools create pd --cluster tidb --machine-type n2-standard-4 --num-nodes=1 \
         --node-labels=dedicated=pd --node-taints=dedicated=pd:NoSchedule
-    gcloud container node-pools create tikv --cluster tidb --machine-type n1-highmem-8 --num-nodes=1 \
+    gcloud container node-pools create tikv --cluster tidb --machine-type n2-highmem-8 --num-nodes=1 \
         --node-labels=dedicated=tikv --node-taints=dedicated=tikv:NoSchedule
-    gcloud container node-pools create tidb --cluster tidb --machine-type n1-standard-8 --num-nodes=1 \
+    gcloud container node-pools create tidb --cluster tidb --machine-type n2-standard-8 --num-nodes=1 \
         --node-labels=dedicated=tidb --node-taints=dedicated=tidb:NoSchedule
     ```
 
     The process might take a few minutes.
 
+## Configure StorageClass
+
+After creating GKE cluster, the cluster contain three StorageClass that are different disk types.
+
+* standard: `pd-standard` disk type (default)
+* standard-rwo: `pd-balanced` disk type
+* premium-rwo: `pd-ssd` disk type (recommended)
+
+To improve I/O write performance, it is recommended to configure `nodelalloc` and `noatime` in the `mountOptions` field of the StorageClass resource.
+
+```yaml
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+# ...
+mountOptions:
+- nodelalloc,noatime
+```
+
+> **Note:**
+>
+> It is not support to configure `nodelalloc` and `noatime` for the default disk type `pd-standard`.
+
+### Use local storage
+
+Use [Zonal Persistent disks](https://cloud.google.com/compute/docs/disks#pdspecs) as a primary production configuration. To simulate bare metal performance, some GCP instance types provide additional [local store volumes](https://cloud.google.com/kubernetes-engine/docs/how-to/persistent-volumes/local-ssd). You can choose such instances for the TiKV node pool to achieve higher IOPS and lower latency.
+
+> **Note:**
+>
+> You cannot dynamically change the storage class of a running TiDB cluster. You can create a new cluster for testing.
+>
+> During the GKE upgrade, [data in the local storage will be lost](https://cloud.google.com/kubernetes-engine/docs/how-to/persistent-volumes/local-ssd) due to the node reconstruction. When the node reconstruction occurs, you need to migrate data in TiKV. If you do not want to migrate data, it is recommended not to use the local disk in the production environment.
```

Comment:
```suggestion
> You cannot dynamically change StorageClass for a running TiDB cluster. For testing purposes, create a new TiDB cluster with the desired StorageClass.
>
> GKE upgrade might cause node reconstruction. In such cases, [data in the local storage might be lost](https://cloud.google.com/kubernetes-engine/docs/how-to/persistent-volumes/local-ssd). To avoid data loss, you need to back up TiKV data before node reconstruction. It is thus not recommended to use local disks in the production environment.
```